contribs/gmf/apps/mobile/index.html

```diff
@@ -30,6 +30,10 @@
         ng-click="mainCtrl.toggleRightNavVisibility()">
       </button>
       <div class="overlay" ng-click="mainCtrl.hideNav()"></div>
+      <button class="ngeo-mobile-geolocation" ngeo-mobile-geolocation="::"
```

Comment:
I'm not sure I understand.

For the `"::"`, I thought that avoided the two-way binding.  What's the correct syntax ?